### PR TITLE
Typo, code cosmetics, duplicate jquery selectors

### PR DESCRIPTION
**Pull request by DatDraggy**
Hope ya don't mind me creating a pull request

- readme typo, Englisch to English
- Removed stray <a> in german files
- Alt attribute for flags
- Align is deprecated, use style
- a can't be disabled
- removed duplicate jquery selectors

### CODE REVIEW

File: de/reservation-form.html [hunk 1]
```diff
@@ -408,7 +408,7 @@ <h3>Zimmertyp</h3>
 
 
           <div class="ef-formgroup form-group">
-            <h3>Kommentare</h3>
+            <h3><label for="comments">Kommentare</label></h3>
```
Comment (dogpixels) [inline de/reservation-form.html:488]: This addition doesn't seem to have the desired effect (Firefox 69.0.1).

Comment (DatDraggy) [reply de/reservation-form.html:488]: I'd appreciate if you could elaborate and maybe send a screenshot. I could not notice a difference besides better accessibility and a better filled out element around the textarea.

Comment (DatDraggy) [reply de/reservation-form.html:488]: Maybe it's a result of the dependencies not being linked anywhere. I have no idea if the files I found are correct, as the site does look a bit scuffed. Would be nice to have a better explanation than just the .gitignore.

**Review by dogpixels — CHANGES_REQUESTED** [review]
Let's wrap this PR up. I'd like to suggest to keep the _disabled_ attribute on both instances of anchor tags, as it enables some unmistakable styling per bootstrap definition that lets the user know this button-styled link is not to be clicked. The PR will be merged within two more days regardless of whether it has been choosen to leave or remove the _disabled_ attribute. 
Thank you for your contribution and we we hope to see more of you!

**Comment by DatDraggy** [comment]
I agree, that is a good idea, however, imo it should not be an anchor, but a button instead. Disabling an anchor is invalid and can mess with/confuse accessibility devices afaik

What about this?

**Comment by dogpixels** [comment]
Perfect!

**Pull request merged**